### ISSUE TITLE
Errors after login in tty

### ISSUE THREAD

**Issue by JSamir** [tomyun/base16-fish#4]
Hi,

after I login in tty, I get some errors:
```
~/.config/fish/config.fish (line 43):
  function put_template: test $1 -lt 16 && printf "\e]P%x%s" $1 (echo $2 | sed 's/\///g'); end; 
....
```

I did not include everything because I needed to type it myself :)

Everything works finde after I start x and use a terminal, but it is still annoying to see this error after login in tty.

**Comment by ngmoviedo** [comment]
Can confirm. There are a lot of errors, but all of them lead to that line. It complains about the utilization of function test, but I haven't been able to determine what's wrong there.

**Comment by ccryx** [comment]
The reason is the usage of $1 and $2 in line 43 of the respective template (base16-oceanicnext.fish in my case). These are not valid in fish and using $argv[1] and $argv[2] solves the issue and makes it actually work. I'm going to create a PR but in the meantime you can change it yourself.

**Comment by ngmoviedo** [comment]
I've just created a PR following @ccryx's advice. All files have been corrected.

**Comment by tomyun** [comment]
Hopefully it's solved by #5 which just got merged.
